tests/test_logging.py

```diff
@@ -43,9 +51,68 @@ def test_get_logger_log_levels(given, expected):
     assert logger.getEffectiveLevel() == expected
 
 def test_multiple_log_handlers():
-    logger = cdislogging.get_logger('one_handler')
+    logger = cdislogging.get_logger('one_handler', log_level='debug')
     assert len(logger.handlers) == 1
 
     # make sure it only has one handler associated with the logger name
-    logger = cdislogging.get_logger('one_handler')
+    logger = cdislogging.get_logger('one_handler', log_level='debug')
     assert len(logger.handlers) == 1
+
+def test_inheritance():
```

Comment:
First: yay unit tests!! :D second, I think we can break this test up into smaller tests. every time you try to log something new and assert something else could be a new test. I think it'd improve the readability a bit to chop this into pieces and add some docstrings for the tests